### ISSUE TITLE
Language match for Pahari

### ISSUE THREAD

**Issue by olcsheehan** [D-PLACE/dplace-data#330]
Hello all,

I think the language match for Pahari (Ef7, xd669) is incorrect. Currently this society is matched to Mahasu Pahari (maha1287). However, Berremann (1963), which is the main source for this society, states explicitly that the people of Sirkanda (to whom this society was coded ‘with special reference’) speak Gahrwali (gahr1243).

Best wishes,

Oliver Sheehan

**Comment by hrncirv** [comment]
Yes, it should be changed, but it is spelled Garhwali (garh1243). https://glottolog.org/resource/languoid/id/garh1243

**Comment by olcsheehan** [comment]
Yes, you're right
